utilities/transactions/lock/point/point_lock_mgr.h

```diff
@@ -17,41 +17,28 @@
 #include "util/autovector.h"
 #include "util/hash_map.h"
 #include "util/thread_local.h"
+#include "utilities/transactions/lock/lock_mgr.h"
+#include "utilities/transactions/lock/lock_tracker.h"
+#include "utilities/transactions/lock/point/point_lock_tracker.h"
 #include "utilities/transactions/pessimistic_transaction.h"
 
 namespace ROCKSDB_NAMESPACE {
 
-class ColumnFamilyHandle;
 struct LockInfo;
 struct LockMap;
 struct LockMapStripe;
 
-struct DeadlockInfoBuffer {
- private:
-  std::vector<DeadlockPath> paths_buffer_;
-  uint32_t buffer_idx_;
-  std::mutex paths_buffer_mutex_;
-  std::vector<DeadlockPath> Normalize();
-
- public:
-  explicit DeadlockInfoBuffer(uint32_t n_latest_dlocks)
-      : paths_buffer_(n_latest_dlocks), buffer_idx_(0) {}
-  void AddNewPath(DeadlockPath path);
-  void Resize(uint32_t target_size);
-  std::vector<DeadlockPath> PrepareBuffer();
-};
-
 struct TrackedTrxInfo {
   autovector<TransactionID> m_neighbors;
   uint32_t m_cf_id;
   bool m_exclusive;
   std::string m_waiting_key;
 };
 
-class Slice;
 class PessimisticTransactionDB;
 
-class TransactionLockMgr {
+// Point lock manager
+class TransactionLockMgr : public BaseLockMgr {
```

Comment:
can we rename it to `PointLockManager` to be consistent with namings like `PointLockTracker`?